src/Umbraco.Commerce.Checkout/Pipeline/Tasks/CreateUmbracoCommerceCheckoutDataTypesTask.cs

```diff
@@ -29,88 +31,107 @@ public class CreateUmbracoCommerceCheckoutDataTypesTask : PipelineTaskBase<Insta
         public override PipelineResult<InstallPipelineContext> Execute(PipelineArgs<InstallPipelineContext> args)
```

Comment:
Lets make these execute methods async so that we don't have to `GetAwaiter().GetResult()` everywhere.